### PR TITLE
Add support for windows platform

### PR DESCRIPTION
**Pull request by SemenchenkoVitaliy**

### CODE REVIEW

**Comment by SemenchenkoVitaliy** [comment]
ping @lundibundi @nechaido

**Review by lundibundi — APPROVED** [review]

File: lib/filestorage.js [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 const fs = require('fs');
 const path = require('path');
-const common = require('metarhia-common');
-const utils = require('./utils.js');
+const common = require('@metarhia/common');
```
Comment (nechaido) [inline lib/filestorage.js:5]: I'd prefer to move into separate commit.

**Review by nechaido — APPROVED** [review]

**Comment by SemenchenkoVitaliy** [comment]
Landed in 6535cd27bbefa61a7dc8ae4a16282ec906edcfeb and bd29b047f5c7b1dd50af5116c7a727dcb9ff60ff.
